packages/roosterjs-editor-dom/lib/htmlSanitizer/HtmlSanitizer.ts

```diff
@@ -181,8 +187,39 @@ export default class HtmlSanitizer {
         const isFragment = nodeType == NodeType.DocumentFragment;
 
         let element = <HTMLElement>node;
+        let shouldKeep: boolean;
+
+        if (isElement) {
+            const tag = getTagOfNode(node);
+            const callback = this.elementCallbacks[tag];
+            if (callback) {
+                shouldKeep = callback(node as HTMLElement, context);
+            } else if (this.allowedTags.indexOf(tag) >= 0 || tag.indexOf(':') > 0) {
+                shouldKeep = true;
+            } else if (this.disallowedTags.indexOf(tag) >= 0) {
+                shouldKeep = false;
+            } else if (this.unknownTagReplacement === '') {
+                shouldKeep = true;
```

Comment:
Having an unknownTagReplacement with an empty string, wouldn't the expected outcome be that the tag is stripped? Maybe it could be safer to have a specific string that said something like "keep" be less confusing? Maybe it is a common pattern that I am just not familiar with, but just a suggestion.

Reply:
+1, I also thought that empty string meant drop the unknown tags as I didn’t think there will be something different between null/undefined and empty string. Then I read the description and realized they are different.

Reply:
Maybe one alternative can be

```
type UnknownTagHandlingStrategy = 
  | { strategy: ‘keep’ }
  | { strategy: ‘drop’ }
  | { strategy: ‘replace’, replacement: string }
```

Reply:
How about use "*" to keep tags, and other valid string to replace, and falsy (null, undefined, empty string) to drop?